modules/tagsparser/tags-parser-grammar.ym

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2002-2015 Balabit
```

Comment:
The year in copyright can be updated to 2017. The other copyright sections look ok.